virtool/startup.py

```diff
@@ -405,4 +407,5 @@ async def startup_tasks(app: Application):
     await tasks_data.create(CleanReferencesTask)
     await tasks_data.create(TimeoutJobsTask)
 
+    await tasks_data.create_periodically(PromoteAdministratorsTask, 600)
```

Comment:
No need to add this here, this method of spawning tasks will be deprecated and removed in a later version.